### PR TITLE
Create sqlite without spectrum

### PR DESCRIPTION
**Pull request by niekdejonge**
Currently the spectrum ids from the metadata are used for creating new libraries.
These are available for GNPS spectra, but this creates an issue, when you want to add your own spectra to the library. 
Solution create integers as spectrum ids, instead of using the ones in the metadata.

### CODE REVIEW

**Comment by niekdejonge** [comment]
@florian-huber This pull request changes the way library spectra are stored. Instead using the GNPS spectrum ids, now integers will be generated that are used as spectrum id. Could you maybe do a review to make sure I am not breaking anything by accident?

File: ms2query/library_files_creator.py [hunk 1]
```diff
@@ -133,11 +133,11 @@ def _load_spectra_and_minimal_processing(self,
         """
         # Loads the spectra from a pickled file
         list_of_spectra = load_pickled_file(pickled_spectra_file_name)
-        assert list_of_spectra[0].get(self.settings[
-                                          "spectrum_id_column_name"]), \
-            f"Expected spectra to have '" \
-            f"{self.settings['spectrum_id_column_name']}' in " \
-            f"metadata, to solve specify the correct spectrum_solumn_name"
+        # assert list_of_spectra[0].get(self.settings[
```
Comment (florian-huber) [inline ms2query/library_files_creator.py:136]: Why comment out?
I would either keep or remove this.

File: ms2query/library_files_creator.py [hunk 2]
```diff
@@ -186,7 +186,8 @@ def create_all_library_files(self,
             self.list_of_spectra,
             columns_dict={"precursor_mz": "REAL"},
             progress_bars=self.settings["progress_bars"],
-            spectrum_id_column_name=self.settings["spectrum_id_column_name"])
+            # spectrum_id_column_name=self.settings["spectrum_id_column_name"]
```
Comment (florian-huber) [inline ms2query/library_files_creator.py:189]: Same here: keep or remove.

File: ms2query/library_files_creator.py [hunk 3]
```diff
@@ -218,8 +219,7 @@ def store_ms2ds_embeddings(self,
 
         # Compute spectral embeddings
         embeddings = ms2ds.calculate_vectors(self.list_of_spectra)
-        spectrum_ids = [s.get(self.settings["spectrum_id_column_name"])
-                        for s in self.list_of_spectra]
+        spectrum_ids = list(range(len(self.list_of_spectra)))
```
Comment (florian-huber) [inline ms2query/library_files_creator.py:222]:
```suggestion
        spectrum_ids = np.arange(0, len(self.list_of_spectra))
```

**Review by florian-huber — CHANGES_REQUESTED** [review]
Looks good Niek!
Just a few minor code cleaning things in my suggestions.

**Pull request merged**